packages/x-charts-pro/src/ChartsToolbarPro/ChartsToolbarPro.tsx

```diff
@@ -39,9 +65,68 @@ export function ChartsToolbarPro(props: ChartsToolbarProps) {
     );
   }
 
+  if (showExportMenu) {
+    const Tooltip = slots.baseTooltip;
+    const MenuList = slots.baseMenuList;
+    const MenuItem = slots.baseMenuItem;
+    const ExportIcon = slots.exportIcon;
+
+    const closeExportMenu = () => setExportMenuOpen(false);
+
+    children.push(
+      <React.Fragment key="export-menu">
+        <Tooltip title={localeText.toolbarExport}>
+          <ToolbarButton
+            ref={exportMenuTriggerRef}
+            id={exportMenuTriggerId}
+            aria-controls={exportMenuId}
+            aria-haspopup="true"
+            aria-expanded={exportMenuOpen ? 'true' : undefined}
+            onClick={() => setExportMenuOpen(!exportMenuOpen)}
+          >
+            <ExportIcon />
+          </ToolbarButton>
+        </Tooltip>
+
+        <ChartsMenu
+          target={exportMenuTriggerRef.current}
+          open={exportMenuOpen}
+          onClose={closeExportMenu}
+          position="bottom-end"
+        >
+          <MenuList
+            id={exportMenuId}
+            aria-labelledby={exportMenuTriggerId}
+            autoFocusItem
+            {...slotProps?.baseMenuList}
+          >
+            {!printOptions?.disableToolbarButton && (
+              <ChartsToolbarPrintExportTrigger
+                render={<MenuItem {...slotProps?.baseMenuItem} />}
+                options={printOptions}
+                onClick={closeExportMenu}
+              >
+                {localeText.toolbarExportPrint}
+              </ChartsToolbarPrintExportTrigger>
+            )}
+            {!imageExportOptions?.disableToolbarButton && (
+              <ChartsToolbarImageExportTrigger
```

Comment:
TODO: How to handle the different formats we support? Accept an array for image export options?